flink-python/pyflink/table/tests/test_expression.py

```diff
@@ -19,7 +19,7 @@
 
 from pyflink.table import DataTypes
 from pyflink.table.expression import TimeIntervalUnit, TimePointUnit, JsonExistsOnError, \
-    JsonValueOnEmptyOrError
+    JsonValueOnEmptyOrError, JsonType
```

Comment:
is `JsonType` under `pyflink.table.expression` in Python correct?

Reply:
That's where all other enums are as well. Or am I misunderstanding the question?

Reply:
isn't this a mistake actually? but let's leave it as it is for now.